lib/puppet/provider/selinux_fcontext/semanage.rb

```diff
@@ -30,7 +30,7 @@ def self.prefetch(resources)
     execpipe("#{command(:semanage)} fcontext -lC") do |out|
       lines = out.readlines[2..-1] || []
       lines.each do |line|
-        filespec, filetype, context = line.split(/\s\s+/)
+        filespec, filetype, context = line.scan(/^(\S+)\s+(\S+\s?\S+)\s+(\S+)\s+$/).flatten
```

Comment:
Fine! I based on my experience with a _match.captures_ which with _$_ at the end of regexp did not work properly, but I see now (tested on Vagrant) that you solution is better! Thank you!